service/src/main/java/com/commercetools/pspadapter/payone/mapping/BankTransferInAdvanceRequestFactory.java

```diff
@@ -31,9 +36,19 @@ public BankTransferInAdvanceRequest createPreauthorizationRequest(@Nonnull Payme
         return createRequestInternal(RequestType.PREAUTHORIZATION, paymentWithCartLike, BankTransferInAdvanceRequest::new);
     }
 
+    /**
+     * <b>NOTE:</b> this is a potentially dangerous transaction type for this payment method. See more in
+     * {@link com.commercetools.pspadapter.payone.transaction.paymentinadvance.BankTransferInAdvanceChargeTransactionExecutor BankTransferInAdvanceChargeTransactionExecutor}
+     * @see com.commercetools.pspadapter.payone.transaction.paymentinadvance.BankTransferInAdvanceChargeTransactionExecutor
+     */
     @Nonnull
     @Override
     public BankTransferInAdvanceRequest createAuthorizationRequest(@Nonnull PaymentWithCartLike paymentWithCartLike) {
+        LOG.warn("Unsupported transaction type \"{}\" for payment method \"\".\n" +
+                        "\t\tNote: this payment/transaction type officially isn't supported by Payone " +
+                        "and thus the behavior of such transaction handling is undefined.\n" +
+                        "Either change the transaction type or update the service if the Payone API has changed.",
+                RequestType.AUTHORIZATION, BANK_TRANSFER_ADVANCE.getKey());
```

Comment:
I see only one `{}`. How does `BANK_TRANSFER_ADVANCE.getKey()` work?

Reply:
nice catch, I think its missing out between the escaped quotes here: `or payment method \"\"`, right @andrii-kovalenko-ct ?